lib/march_hare/channel.rb

```diff
@@ -614,10 +614,14 @@ def basic_get(queue, auto_ack)
       end
     end
 
-    def basic_consume(queue, auto_ack, consumer)
```

Comment:
This is a breaking API change but OK, we'll bump the version as it's better than any of the alternatives I can think of.